tests/path_gateway_dag_test.go

```diff
@@ -231,86 +232,121 @@ func TestPlainCodec(t *testing.T) {
 		plainCID := plain.Cid()
 		plainOrDagCID := plainOrDag.Cid()
 
-		tests := SugarTests{
-			{
-				Name: Fmt(`GET {{name}} without Accept or format= has expected "{{format}}" Content-Type and body as-is`, row.Name, row.Format),
-				Hint: `
+		var dagFormattedResponse []byte
+
+		tests := &SugarTests{}
```

Comment:
Could you stick to values?

(details in https://github.com/ipfs/gateway-conformance/pull/162/files#r1333946686)